tests/e2e/jest.config.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+  ...require('@wordpress/scripts/config/jest-e2e.config'),
+  transform: {
+    '^.+\\.[jt]sx?$':
+      '<rootDir>/../../node_modules/@wordpress/scripts/config/babel-transform',
+  },
+  transformIgnorePatterns: ['node_modules'],
+  setupFilesAfterEnv: [
+    '<rootDir>/config/bootstrap.js',
+    '@wordpress/jest-puppeteer-axe',
+    '@wordpress/jest-console',
+    'expect-puppeteer',
+  ],
+  testPathIgnorePatterns: [
+    '<rootDir>/.git',
+    '<rootDir>/node_modules',
+    '<rootDir>/build',
+    '<rootDir>/tests/e2e/specs/edit-story',
+  ],
+  reporters: [['jest-silent-reporter', { useDots: true }]],
```

Comment:
Wondering if we should also show warnings. Or would it not be silent enough then? :) Thoughts?

Reply:
Let me test. If there are no warnings at the moment we can opt in to show them. If there are some, we should probably fix them.

Reply:
Done in 2c93d513c8980ace7a3925847be4755fbdb4ab84. Seems to work fine.